Smoothing out AWS Getting Started flow (#4678)
pulumi/docs: content/docs/get-started/aws/modify-program.md

$ mkdir site
```

> Next, create a new `index.html` file with some content you will upload to your bucket.

Reviewer: Not your PR, but `with some content you will upload` is a bit awkard. Suggest maybe:

```suggestion
Next, create a new `index.html` file containing a bit of content.
```

.. or the like.

{{< chooser os "macos,linux,windows" / >}}

{{% choosable os macos %}}

```bash
$ cat <<EOT > site/index.html
<html>
    <body>
        <h1>Hello, Pulumi!</h1>
    </body>
</html>
EOT
```

{{% /choosable %}}

{{% choosable os linux %}}

```bash
$ cat <<EOT > site/index.html
<html>
    <body>
        <h1>Hello, Pulumi!</h1>
    </body>
</html>
EOT
```

{{% /choosable %}}

{{% choosable os windows %}}

```bat
> site/index.html (
    @echo.^<html^>
    @echo.  ^<body^>
    @echo.      ^<h1^>Hello, Pulumi!^</h1^>
    @echo.  ^</body^>

>     @echo.^</html^>)

Reviewer 2: Not a huge deal - but these may make the prose slightly harder to use in IDE-like environments (vs. directly in a shell) - especially this Windows one which can't be pasted into an IDE/editor.  I wonder if simplifying this in the other direction would be better "Create a file named `site/index.html` and add the following text".

Author: It feels like we should be okay here because the preceding steps in the tutorial assume people are in a shell to use the Pulumi CLI, make directories, etc.

Btw, I have updated the text below this section to make sure folks know to transition to an IDE or text editor.
